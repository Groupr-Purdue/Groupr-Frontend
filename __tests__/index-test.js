import React from 'react';
import renderer from 'react-test-renderer';
test('jest snapshot test', () => {
  let tree = renderer.create(<div> goodbye </div>).toJSON();
  expect(tree).toMatchSnapshot();
});
