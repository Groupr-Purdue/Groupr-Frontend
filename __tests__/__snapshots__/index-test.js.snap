exports[`test jest snapshot test 1`] = `
<div>
   goodbye 
</div>
`;
